### PR TITLE
10710 improve code quality checks flow in git hooks

### PR DESCRIPTION
**Pull request by andrei-kucherov**

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.4
+lockfileVersion: 5.3
```
Comment (Akiyamka) [inline pnpm-lock.yaml]: let's update our local pnpms!

**Review by Akiyamka — COMMENTED** [review]
i  recomend update package.lock too

**Review by Akiyamka — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -28,13 +28,13 @@
     "prepare": "node ./scripts/husky-prepare.js"
   },
   "lint-staged": {
-    "*/**/*.{ts, tsx}": [
-      "eslint --fix --cache"
+    "**/*.{ts,tsx}": [
+      "eslint --cache --fix --quiet --rule 'no-console: error'"
```
Comment (propakov) [inline package.json]: @andrei-kucherov it doesn't fix eslint warnings with --quiet, i've just fixed it recently
tried with wrong imports order

Comment (andrei-kucherov) [reply package.json]: agree. fixed

**Review by propakov — APPROVED** [review]
good job! 🚀

**Pull request merged**